### PR TITLE
Add hotfix tags for hyperkube image

### PR DESCRIPTION
**Pull request by nilo19**
Add hotfix tags for hyperkube image

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -472,15 +472,19 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.11-hotfix.20200529.1
 1.15.11-hotfix.20200714.1
 1.15.11-hotfix.20200817.1
+1.15.11-hotfix.20200824
```
Comment (sozercan) [inline vhdbuilder/packer/install-dependencies.sh:475]: Just base image patched hyperkube for `hotfix.20200824`, can you add `.1` to all of `hotfix.20200824` images?

**Review by feiskyer — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -472,15 +472,19 @@ PATCHED_HYPERKUBE_IMAGES="
...
 1.15.12-hotfix.20200623.1
 1.15.12-hotfix.20200714.1
 1.15.12-hotfix.20200817.1
+1.15.12-hotfix.20200824
```
[the remark this answers is not in the document]
Comment (nilo19) [reply vhdbuilder/packer/install-dependencies.sh]: the hotfix is about kube-controller-maneger, why is kubectl/kubelet needed to be rebuilt?

[the remark this answers is not in the document]
Comment (nilo19) [reply vhdbuilder/packer/install-dependencies.sh]: updated

File: vhdbuilder/packer/install-dependencies.sh [hunk 3]
```diff
@@ -416,13 +416,18 @@ K8S_VERSIONS="
...
+1.15.12-hotfix.20200824.1
 1.16.9-hotfix.20200529.1
 1.16.10-hotfix.20200817.1
+1.16.10-hotfix.20200824.1
 1.16.13-hotfix.20200817.1
+1.16.13-hotfix.20200824.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200817.1
 1.17.9-hotfix.20200817.1
+1.17.9-hotfix.20200824.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:430]: for kubelet and kubectl we just need the latest version on the node. It's different than hyperkube that is decided by ccp charts

Comment (nilo19) [reply vhdbuilder/packer/install-dependencies.sh]: got it, updated

File: vhdbuilder/packer/install-dependencies.sh [hunk 4]
```diff
@@ -472,22 +472,27 @@ PATCHED_HYPERKUBE_IMAGES="
...
 1.16.10-hotfix.20200817.1
+1.16.10-hotfix.20200824.1
 1.16.13-hotfix.20200714.1
 1.16.13-hotfix.20200817.1
+1.16.13-hotfix.20200824.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200624
 1.17.7-hotfix.20200714.1
 1.17.7-hotfix.20200714.2
 1.17.7-hotfix.20200817.1
```
Comment (sozercan) [inline vhdbuilder/packer/install-dependencies.sh:492]: we should not use `hyperkube:v1.17.7-hotfix.20200817.1` as this one has iptables issue

Comment (sozercan) [reply vhdbuilder/packer/install-dependencies.sh:492]: If hyperkube v1.17.7 needs to be there, `v1.17.7-hotfix.20200714.1` does not have this issue

Comment (nilo19) [reply vhdbuilder/packer/install-dependencies.sh]: deleted

File: vhdbuilder/packer/install-dependencies.sh [hunk 5]
```diff
@@ -472,22 +472,26 @@ PATCHED_HYPERKUBE_IMAGES="
...
 1.16.10-hotfix.20200817.1
+1.16.10-hotfix.20200824.1
 1.16.13-hotfix.20200714.1
 1.16.13-hotfix.20200817.1
+1.16.13-hotfix.20200824.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200624
 1.17.7-hotfix.20200714.1
 1.17.7-hotfix.20200714.2
-1.17.7-hotfix.20200817.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:488]: why do we remove this?

Comment (sozercan) [reply vhdbuilder/packer/install-dependencies.sh:488]: https://github.com/Azure/AgentBaker/pull/263#discussion_r478041412

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:488]: thanks @sozercan !

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:488]: #271  to catch future issue

**Pull request merged**